flink-contrib/src/test/java/org/apache/flink/contrib/SimpleTweetInputFormatTest.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.flink.contrib;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.contrib.tweetinputformat.io.SimpleTweetInputFormat;
+import org.apache.flink.contrib.tweetinputformat.model.tweet.Tweet;
+import org.apache.flink.contrib.tweetinputformat.model.tweet.entities.HashTags;
+import org.apache.flink.core.fs.FileInputSplit;
+import org.apache.flink.core.fs.Path;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+public class SimpleTweetInputFormatTest {
+
+    private Tweet tweet;
+
+    private SimpleTweetInputFormat simpleTweetInputFormat;
+
+    private FileInputSplit fileInputSplit;
+
+    protected Configuration config;
+
+    protected File tempFile;
+
+
+    @Before
+    public void testSetUp() {
+
+
+        simpleTweetInputFormat = new SimpleTweetInputFormat();
+
+        File jsonFile = new  File("../flink-contrib/src/main/resources/HashTagTweetSample.json");
+
+        fileInputSplit = new FileInputSplit(0, new Path(jsonFile.getPath()), 0, jsonFile.length(), new String[] {"localhost"});
+    }
+
+    @Test
+    public void testTweetInput() throws Exception {
+
+
+        simpleTweetInputFormat.open(fileInputSplit);
+        List<String> result;
+
+        while (!simpleTweetInputFormat.reachedEnd()) {
+            tweet = new Tweet();
+            tweet = simpleTweetInputFormat.nextRecord(tweet);
+
+            if(tweet != null){
```

Comment:
I think this behavior was intended, by checking the DelimitedInputFormat UnitTest I found this, I will edit my test case, to fix the problem.
![selection_019](https://cloud.githubusercontent.com/assets/2375289/6618391/d3366786-c8c2-11e4-8854-2a38a8da4da3.png)